### PR TITLE
Add Mayhem integration

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,7 @@ minimp3
 minimp3_arm
 *.gcda
 *.gcno
-*.gcov
\ No newline at end of file
+*.gcov
+
+fuzzing/afl*
```
Comment (xansec) [inline .gitignore]: Is there a reason these need to be ignored?

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+project: rnshah9/mayhem-minimp3
+
+target: fuzz
```
Comment (xansec) [inline Mayhemfile]: I might change this to be `fuzz-minimp3`.

File: Mayhemfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+image: ghcr.io/rnshah9/mayhem-minimp3
```
Comment (xansec) [inline Mayhemfile]: You can omit the image field as it gets added during the action step.

File: fuzzing/get-afl.sh [hunk 4]
```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env bash
 cd "${0%/*}"
-rm afl-latest.tgz
```
Comment (xansec) [inline fuzzing/get-afl.sh:3]: I think these all make sense, but a quick summary of why you made these changes to `get-afl.sh` would be helpful.

**Review by xansec — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (rnshah9) [reply .gitignore]: Ah no, that was for my personal usage; I'll delete this.

**Comment by rnshah9** [comment]
Made changes to get-afl.sh:
- `rm afl-latest.tgz` will fail on the first try, so ignore its error output
- Add path to `LLVM_CONFIG` for build
- The AFL build is perfectly functional but returns an error code, so don't early exit if it fails.

**Comment by xansec** [comment]
Thanks! LGTM!

**Pull request merged**